assets/scss/templates/_header.scss

```diff
@@ -239,14 +239,6 @@
     align-items: center;
   }
 
-  span {
-    text-overflow: ellipsis;
-    white-space: nowrap;
-    max-width: 248px;
-    display: block;
-    overflow: hidden;
-  }
-
```

Comment:
Is this code really redundant? @damianmuti please, verify all demo pages in the library and update reference in webapp before merge it.

Reply:
![image](https://user-images.githubusercontent.com/1157864/54716110-c7dbd900-4b33-11e9-94b0-fd9b0a52f6fc.png)

This issue seems to be related to your change, could you verify?

Reply:
Added ellipsis back on the specific elements that needed it. Good catch!